portal-api/src/v1/api.js

```diff
@@ -267,14 +267,33 @@ const saveUtilisationReport = async (reportData, reportPeriod, user, fileInfo) =
   }
 };
 
+/**
+ * Gets utilisation reports for a specific bank. If a report
+ * period is not provided, all reports for that bank are
+ * returned. If a report period is provided, the report
+ * submitted for that report period is returned. Returned
+ * reports are ordered by year descending and month
+ * ascending.
+ * @param {string} bankId
+ * @param {import('../types/utilisation-reports').ReportPeriod} [reportPeriod]
+ */
 const getUtilisationReports = async (bankId, reportPeriod) => {
   try {
     if (!isValidBankId(bankId)) {
       console.error('Get utilisation reports failed with the following bank ID %s', bankId);
       throw new Error('Invalid bank ID provided: %s', bankId);
     }
 
-    if (reportPeriod && !isValidReportPeriod(reportPeriod)) {
+    if (!reportPeriod) {
+      const response = await axios({
+        method: 'get',
+        url: `${DTFS_CENTRAL_API_URL}/v1/bank/${bankId}/utilisation-reports`,
+        headers: headers.central,
+      });
+      return response.data;
+    }
+
+    if (!isValidReportPeriod(reportPeriod)) {
       console.error('Get utilisation reports failed with the following report period:', reportPeriod);
       throw new Error('Invalid report period provided: %s', reportPeriod);
```

Comment:
Passing `reportPeriod` as a second argument to populate the `%s` works for `console.error` but not for the `Error` class. Same issue above for
```
throw new Error('Invalid bank ID provided: %s', bankId);
```